Formula/a/atmos.rb

```diff
@@ -6,6 +6,11 @@ class Atmos < Formula
   license "Apache-2.0"
   head "https://github.com/cloudposse/atmos.git", branch: "main"
 
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:[._]\d+)+)$/i)
+  end
```

Comment:
btw, why not just go with this standard livecheck fmt?
```suggestion
  livecheck do
    url :stable
    regex(/^v?(\d+(?:\.\d+)+)$/i)
  end
```

Reply:
i'm too lazy to write regex from scratch. most of the times i just blindly copy it from similar formulae and adjust only if it doesn't work. thank you for noticing it

btw, can you add `CI-syntax-only` label?